lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -180,6 +197,15 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
                 this.loadProperties(node);
             });
 
+        if (this.displayPredictions) {
+            this.predictionService.predictionStatusUpdated$.pipe(takeUntil(this.onDestroy$)).subscribe(({key, previousValue}) => {
+                this.cardViewContentUpdateService.onPredictionStatusChanged([{key, previousValue}]);
+                this.nodesApiService.getNode(this.node.id).subscribe((node) => {
```

Comment:
Can we unsubscribe this subscription via takeUntil() or pipe(first()) ?

Reply:
done, ty